index.html

```diff
@@ -31,11 +31,11 @@ <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see<
             <div class="lets-connect">LET'S CONNECT</div>
             <div class="lets-connect-icons">
                <ul>
```

Comment:
Web pages often have sections of information separated by visual headings, for example, heading text is bigger and bold. To make these work for everyone, the headings need to be [marked up](https://www.w3.org/WAI/test-evaluate/preliminary/#markup)

Kindly use a header tag `hx` e.g `h2` . . . `h6` for the text `LET'S CONNECT`. so that all text that looks like a heading is marked up as a heading.